client/src/ui/atoms/TextField.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { InputHTMLAttributes } from 'react';
+import Close from '@material-ui/icons/Close';
+
+interface Props extends InputHTMLAttributes<HTMLInputElement> {
+    helperText?: string;
+    id: string;
+    invalid: boolean;
+    labelText: string;
+    placeholder?: string;
+}
+const TextField = ({ id, labelText, placeholder, invalid, helperText, ...rest }: Props): JSX.Element => {
+    return (
```

Comment:
To keep this consistent with other components lets remove the `return`   => (<div></div>)